bc_obps/reporting/tests/service/test_report_activity_save_service/test_save_report_emission.py

```diff
@@ -117,14 +123,27 @@ def test_update_emission(self):
         report_emission = service_under_test.save_emission(
             report_source_type,
             report_fuel,
-            {"test_emission_prop": "something", "gasType": "GGIRCA"},
+            {
+                "test_emission_prop": "something",
+                "gasType": "GGIRCA",
+                "methodology": {"id": 9003, "methodology": "Default HHV/Default EF"},
+            },
         )
-
         make_recipe("reporting.tests.utils.gas_type", chemical_formula="BCOBPS")
         updated_return_value = service_under_test.save_emission(
             report_source_type,
             report_fuel,
-            {"id": report_emission.id, "test_emission_prop": "new something", "gasType": "BCOBPS"},
+            {
+                "id": report_emission.id,
+                "test_emission_prop": "new something",
+                "gasType": "BCOBPS",
+                "methodology": {
+                    "id": 9003,
+                    "methodology": "Default EF",
+                    "unitFuelCo2DefaultEmissionFactor": 3,
+                    "unitFuelCo2DefaultEmissionFactorFieldUnits": "kg/GJ",
+                },
+            },
         )
 
         assert report_emission.id == updated_return_value.id
```

Comment:
we're still missing an assertion/test to make sure the `save_methodology` method was called with the right parameters (see the `test_save_fuel` for an example